Task_4/scripts/app.js

```diff
@@ -0,0 +1,32 @@
+const moduleTasks = (function(){
+    const user = 'Julia Grib';
+
+    function isValidateId(id){
+        return typeof id === 'string' ? true : false;
+    }
+
+    function getTask(id) {
+        if(isValidateId(id)) {
+            return tasks.find(task => task.id === id) || new Error(errors.taskNotFound);
+        }
+        return new Error(errors.invalidValue);
+    }
+
+    function removeTask(id){
+        if(isValidateId(id)){
+            let index = tasks.findIndex(task => (task.id === id) && (task.assignee === user));
```

Comment:
you can use filter() 
tasks = tasks.filter(()=>(task.id !== id) && (task.assignee !== user))